public/docs/_examples/cb-validation/ts/app/hero-form-model-binding.component.html

```diff
@@ -0,0 +1,75 @@
+ <!-- #docplaster -->
+ <!-- #docregion -->
+<div class="container">
+  <div [hidden]="submitted">
+    <h1>Hero Form (Model-Driven with Data Binding)</h1>
+    <form [ngFormModel]="heroForm" *ngIf="active" 
+          (ngSubmit)="onSubmit()">
+      <div class="form-group">
+        <!-- #docregion name-with-error-msg -->
+        <label for="name">Name</label>
+        <input type="text" class="form-control" 
```

Comment:
Noting this here, but it looks like an issue with most of the `input`s - these should have an `id` with the same value as the corresponding `label`'s `for`, or be nested within the `label`.